docs/specs/code/SERVER_STATUS_ENDPOINT.md

```diff
@@ -0,0 +1,37 @@
+# Abstract
+
+Describe a new endpoint to notify server issues in Yoroi-frontend.
+   
+# Motivation
+
+Currently, Yoroi-frontend has no way to notify neither server issues nor connection issues. Also, once Yoroi has the ability to notify server issues, the endpoint used by Yoroi to fetch the information can be used to create a public URL that reports the mentioned status, such as [AWS](https://status.aws.amazon.com/) and [Twitter](https://status.twitterstat.us).
+
+# Proposal
+
+Adding a new endpoint in the backend named `api/status` is proposed. This endpoint has the method `GET` that returns the last JSON inserted in the following table.
+
+```
+CREATE TABLE api_statuses (
+   ID serial NOT NULL PRIMARY KEY,
+   status json NOT NULL
+);
+```
+
+The element insertion will be performed manually into the database.
+
+The members of this JSON are TBD. Tentatively and as stated by [ch1223 conversation](https://app.clubhouse.io/emurgo/story/1223/healthcheck-endpoint), the JSON contains a boolean with the state value, a string with the problem description and a timestamp.
+
+The string with the problem description can be used to be displayed in a banner that Yoroi-frontend will render. However, this comes with the problem of i18n, i.e., showing an English message in all the languages or inserting the problem description translated for all the available languages in the JSON. Considering this, it is preferred to display the same message regardless of the problem description (and have this message translated in `app/i18n`).
+
+#GUI
+
+The GUI component consists of a red banner that will be displayed only if the endpoint reports a problem or the connection is not possible. The banner appearance is similar to the one displayed in the staging network and the message is `WARNING: CONNECTION TO THE SERVER FAILED. PLEASE CHECK OUR TWITTER ACCOUNT https://twitter.com/yoroiwallet/`. For example:
+
+![Server error](../images/banner_example_1.png "Server error")
+
+And when the connection with the endpoint fails the message is `WARNING: CONNECTION TO THE SERVER FAILED. PLEASE CHECK YOUR INTERNET CONNECTION OR OUR TWITTER ACCOUNT https://twitter.com/yoroiwallet/`:
+
+![Network error](../images/banner_example_2.png "Network error")
+
+The behavior is that this new banner will overlap the staging banner if both are present.
```

Comment:
In what screens is going to be shown?

Reply:
I see no reason to avoid showing the banner everywhere

Reply:
okay, I would update the specs with that

Reply:
So we've decided not to make the network error dismissible? I'm totally okay with that decision but just need to make sure we're all in agreement with this.